### ISSUE TITLE
Build LDC LTS as a separate flatpak?

### ISSUE THREAD

**Issue by gasinvein** [flathub/org.freedesktop.Sdk.Extension.ldc#2]
Currently, we use old LDC LTS for bootstrapping. We could submit it as a separate SDK extension and still use it for building LDC. This would drastically reduce build times for LDC. Also maybe someone can make use of LDC LTS for, well, something.

**Comment by Cogitri** [comment]
> We could submit it as a separate SDK extension and still use it for building LDC. This would drastically reduce build times for LDC.

Oh, I had expected this would be cached anyway since we always build the same thing

> Also maybe someone can make use of LDC LTS for, well, something.

To be honest, I doubt it, but it probably wont hurt.

**Comment by gasinvein** [comment]
How should we name the extension? E.g. `.ldc-lts`, or `.ldc-bootstrap`, or else?
> Oh, I had expected this would be cached anyway since we always build the same thing

It's generally cached when you build locally, but Flathub builds everything from scratch.

**Comment by Cogitri** [comment]
I think ldc-bootstrap would work, I think ldc-lts sounds too much like it's meant for end users when they want ldc in almost all cases

**Comment by TingPing** [comment]
Before we complicate things, are build times *actually* a problem?

**Comment by gasinvein** [comment]
Not really, but it still could be much faster. if we don't rebuild old LLVM each time.
Does splitting one flatpak to two add complexity?
